src/platforms/dotnet/guides/winui/index.mdx

```diff
@@ -0,0 +1,133 @@
+---
+title: WinUI 3
```

Comment:
I'd also recommend dropping the version from the title.

Just mention WinUI 3 below if needed, if we don't support WinUI2 (I believe we do, and we call that UWP), we might want to have 2 sections in the doc for v2 and v3

Reply:
Yeah I think WinUI 2 is UWP (which has it's own documentation). You reckon we could/should merge these? At the moment there are two different sets of guidance in this page (JIT and AOT)... I guess if we merge it with UWP then there would be 3 sets of guidance in the one page. I think it might be easier to leave the UWP instructions separate, personally.

Reply:
merge and redirect yeah, that'd be best I think

Reply:
I don't have strong feelings about this either. It seems clear as-is to me, but I see bruno's argument.